docs/wiki/protocol/transaction.md

```diff
@@ -0,0 +1,33 @@
+# How do transactions start, and how are they processed in Ethereum?
+How exactly are transactions initiated, and how are they processed in Ethereum? I will explain this phenomenon with the help of Go Ethereum (Geth) code examples.
+
+## Transaction Creation
+The wallet software takes the user's input and constructs a **transaction object**. A user creates and signs a transaction with their private key. This is usually handled by a wallet or a library such as ether.js, web3js, web3py etc but under the hood the user is making a request to a node using the **Ethereum JSON-RPC API**. This object contains all the necessary fields: nonce, gasPrice, gasLimit, to, address, value and data. User sends this basic items to Ethereum.
```

Comment:
Wav. ECDSA is perfect summary.
## Transaction Creation
The wallet software takes the user's input and constructs a **transaction object**. A user creates and signs a transaction with their private key. This is usually handled by a wallet or a library such as ether.js, web3js, web3py etc but under the hood the user is making a request to a node using the **Ethereum JSON-RPC API**. The Ethereum JSON-RPC API provides a standardized way for the wallet or library to communicate with an Ethereum node.  Ethereum. Before constructing a transaction, the wallet software might use JSON-RPC API calls to check the account's balance,the current nonce for the account, and to obtain the current recommended gas price. With this information, the wallet constructs a transaction object that includes fields like nonce, gasPrice, gasLimit, to (recipient address), value (amount of Ether to send), and data. The user signs the constructed transaction with their private key. Finally, the signed transaction is broadcast to the Ethereum network using the JSON-RPC API call.Here's a basic representation of an unsigned Ethereum transaction in JSON format:

```
{
  "nonce": "0x1",
  "gasPrice": "0x09184e72a000",
  "gasLimit": "0x2710",
  "to": "0x0000000000000000000000000000000000000000",
  "value": "0x0",
  "data": "0x"
}

```
After signing, the transaction is serialized into a raw format, typically a hexadecimal string. Here's an example of what a signed transaction might look like (illustration ):
``` 0xf86c018509184e72a00082710c94000000000000000000000000000000000000000000808401a0f38abf4b5c854...```

This hexadecimal string represents the entire transaction, including the signature. It can be directly submitted to the Ethereum network using the  JSON-RPC API call.